package.json

```diff
@@ -82,7 +82,7 @@
     "vscode:bundle:debugger": "lerna run bundle:debugger",
     "vscode:bundle:extension": "lerna run bundle:extension",
     "vscode:package": "npm run vscode:bundle:debugger && npm run vscode:bundle:extension && lerna run vscode:package --concurrency 1 && npm run reformat",
-    "vsix:install": "find ./packages -name '*.vsix' -exec code --install-extension {} \\;",
+    "vsix:install": "find ./packages -name '*.vsix' -exec code-insiders --install-extension {} \\;",
```

Comment:
if I'm the only one using this for vsix testing figured I'd update instead of always editing it.

Reply:
Hmm - maybe we could have two?  vsix:install and vsix:install:insiders?